<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Test suite:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QToolButton" name="btnSelectTest">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1" rowspan="2">
      <widget class="QCheckBox" name="chBoxUseJSVG">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="rBtnSuiteResvg">
          <property name="text">
           <string>resvg</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rBtnSuiteCustom">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Custom</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="6" column="1">
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
      <item row="12" column="0" rowspan="2">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>JSVG:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" rowspan="2" colspan="3">
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </item>
     <item row="10" column="2" rowspan="2">
      <widget class="QLineEdit" name="lineEditBatik">
       <property name="placeholderText">
        <string>Path to batik-rasterizer</string>
       </property>
      </widget>
     </item>
     <item row="12" column="2" rowspan="2">
      <widget class="QLineEdit" name="lineEditJSVG">
       <property name="placeholderText">
        <string>Path to jsvg-rasterizer</string>
       </property>
      </widget>
     </item>
     <item row="10" column="3" rowspan="2">
      <widget class="QToolButton" name="btnSelectBatik">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0" rowspan="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Batik:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" rowspan="2">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>SVGSalamander:</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0" rowspan="2">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>EchoSVG:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2" rowspan="2">
      <widget class="QLineEdit" name="lineEditSVGSalamander">
       <property name="placeholderText">
        <string>Path to svgsalamander-rasterizer</string>
       </property>
      </widget>
     </item>
     <item row="14" column="2" rowspan="2">
      <widget class="QLineEdit" name="lineEditEchoSVG">
       <property name="placeholderText">
        <string>Path to echosvg-rasterizer</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1" rowspan="2">
      <widget class="QCheckBox" name="chBoxUseBatik">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="12" column="3" rowspan="2">
      <widget class="QToolButton" name="btnSelectJSVG">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEditTestsPath"/>
     </item>
     <item row="2" column="1">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="8" column="1" rowspan="2">
      <widget class="QCheckBox" name="chBoxUseSVGSalamander">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="14" column="1" rowspan="2">
      <widget class="QCheckBox" name="chBoxUseEchoSVG">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
      <item row="8" column="3" rowspan="2">
      <widget class="QToolButton" name="btnSelectSVGSalamander">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="14" column="3" rowspan="2">
      <widget class="QToolButton" name="btnSelectEchoSVG">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
